deploy/resources.go

```diff
@@ -4,3 +4,9 @@ import "embed"
 
 //go:embed templates/toolchaincluster/*
 var ToolchainClusterTemplateFS embed.FS
+
+//go:embed autoscaler/*
```

Comment:
minor - should we move these to the templates folder as the ones above ?

Reply:
good point done